### ISSUE TITLE
Dragging causes Microsoft Edge v18 to crash

### ISSUE THREAD

**Issue by csuermann** [bpmn-io/bpmn-js#938]
__Describe the Bug__

When I try to drag a BPMN element on the canvas using the Microsoft Edge browser in the current latest version (18), the browser crashes and reloads the whole page.

__Steps to Reproduce__

1. open https://demo.bpmn.io/s/start in Microsoft Edge browser v18
2. click on a bpmn element on the canvas and try to drag it
3. Notice that the browser tab reloads completely without any visible error.

__Expected Behavior__

The element schould move as I drag it over the canvas

__Environment__

 - Browser: Microsoft Edge 18 (works in 17)
 - OS: Windows 10
 - Library version: any (tried 3.2.0 down to 2.0.0)

**Comment by nikku** [comment]
__Triage Log__

* Blows up [right here](https://github.com/bpmn-io/diagram-js/blob/master/lib/features/preview-support/PreviewSupport.js#L52) attempting to clone the SVG node in order to display the drag preview.

**Comment by nikku** [comment]
No related issue seems to be found in the [MS Edge bug tracker](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/).

**Comment by nikku** [comment]
Fixed in `v3.2.1`.

**Comment by nikku** [comment]
Details on the underlying MS Edge bug (cf. [fiddle, blowing up MS Edge 18](https://jsfiddle.net/Lem921d3/1/)):

```html
<html>
<svg>
  <g></g>
</svg>
</html>
```

```javascript
var g = document.querySelector('g');
var svg = g.ownerSVGElement;

var transformList = g.transform.baseVal;

var transform = g.ownerSVGElement.createSVGTransform();

transform.setTranslate(100, 100);

transformList.initialize(transform);

// MS Edge (17, 18): at this point actual pixel values are stored
// in the SVG. This is invalid:
// 
// <g transform="translate(100px, 100px)" />
//
// To verify: 
// 

console.log('transform =', g.getAttribute('transform'));

// later down the line MS Edge 18 will blow up completely, trying to clone 
// the node:

var clone = g.cloneNode(true);
```

**Comment by nikku** [comment]
Upstream fix in `tiny-svg@2.2.1`: https://github.com/bpmn-io/tiny-svg/commit/980e9d6f69a79ae500c6a4172d046b2420e4ca25
